h2o-docs/src/product/performance-and-prediction.rst

```diff
@@ -2984,6 +2984,148 @@ For classification problems, when running ``h2o.predict()`` or ``.predict()``, t
+        275    0.018293     0.089615               0.018293                0.509146   3
+        276    0.012195     0.073044               0.012195                0.506098   2
+        277    0.006098     0.051560               0.006098                0.503049   1
+        278    0.000000     0.000000               0.000000                0.500000   0
+            fns fps tps      tnr      fnr      fpr      tpr idx
+        273   0 159 126 0.030488 0.000000 0.969512 1.000000 272
+        274   0 160 126 0.024390 0.000000 0.975610 1.000000 273
+        275   0 161 126 0.018293 0.000000 0.981707 1.000000 274
+        276   0 162 126 0.012195 0.000000 0.987805 1.000000 275
+        277   0 163 126 0.006098 0.000000 0.993902 1.000000 276
+        278   0 164 126 0.000000 0.000000 1.000000 1.000000 277
+
+
+    .. code-tab:: python
+
+        # Retrieve your default threshold:
+        original_threshold = model.default_threshold()
+        origininal_threshold
+        0.3343532308872656
+
+        # Threshold is important to decide final prediction. Model prediction output 
+        # is a real number between 0-1 and the threshold is used to decide if the 
+        # output will be 0 or 1. Threshold is decided by max F1 metric by default, but
+        # you can reset the threshold:
+        from h2o.utils.model_utils import reset_model_threshold
+        new_threshold = 0.6917189903
+        old_returned = reset_model_threshold(model, new_threshold)
+        reset_model = h2o.get_model(model.model_id)
+        reset_threshold = reset_model.default_threshold()
+        # Predict with the reset model:
+        predict_reset = reset_model.predict(test)
+        # Compare the predictions:
+        predict.as_data_frame()
+            predict        p0        p1
+        0         0  0.726079  0.273921
+        1         0  0.819360  0.180640
+        2         1  0.340055  0.659945
+        3         0  0.985305  0.014695
+        4         0  0.941166  0.058834
+        ..      ...       ...       ...
+        85        0  0.989945  0.010055
+        86        0  0.915216  0.084784
+        87        0  0.802202  0.197798
+        88        0  0.893146  0.106854
+        89        0  0.697517  0.302483
+
+        [90 rows x 3 columns] 
+        predict_reset.as_data_frame()
+             predict        p0        p1
+        0         0  0.726079  0.273921
+        1         0  0.819360  0.180640
+        2         0  0.340055  0.659945
+        3         0  0.985305  0.014695
+        4         0  0.941166  0.058834
+        ..      ...       ...       ...
+        85        0  0.989945  0.010055
+        86        0  0.915216  0.084784
+        87        0  0.802202  0.197798
+        88        0  0.893146  0.106854
+        89        0  0.697517  0.302483
+
+        [90 rows x 3 columns]
+
+
+        # You can also get optimal thresholds for calculated metrics:
+        model.thresholds_and_metric_scores()
```

Comment:
@hannah-tillman Could you also add the print of `model.thresholds_and_metric_scores()` as in the R example? Thanks!